crates/router/src/compatibility/stripe/customers.rs

```diff
@@ -168,10 +168,8 @@ pub async fn list_customer_payment_method_api(
     state: web::Data<routes::AppState>,
     req: HttpRequest,
     path: web::Path<String>,
-    json_payload: web::Query<payment_methods::PaymentMethodListRequest>,
```

Comment:
Since there are 2 api routes now in core. Can this changes be undo. Make sure it is stripe compatible. https://stripe.com/docs/api/payment_methods/customer_list

Reply:
done